pipeline/metadata/caida_ip_metadata.py

```diff
@@ -231,7 +231,11 @@ def lookup(self, ip: str) -> CaidaReturnValues:
     Raises:
       KeyError: when the IP's ASN can't be found
     """
-    asn, netblock = self.asn_db.lookup(ip)
+    try:
+      asn, netblock = self.asn_db.lookup(ip)
```

Comment:
When things are called "ip", they ought to be an IP. What value did you get?
Perhaps the CP data should rename the field to "answer".
It could also be helpful to be passing around the "ipaddress" type instead of strings (same for date/time).

Reply:
I don't know what the bad values are! We should be able to see them in the new warning logs of the [test backfill](https://console.cloud.google.com/dataflow/jobs/us-central1/2022-05-20_04_57_32-4516889685021138454?project=firehook-censoredplanet) once it gets far enough. I'm assuming it's just some kind of network noise.

We've talked about date string -> date in the past, but it was difficult because beam is fussy about what things can be keys.

Reply:
![image](https://user-images.githubusercontent.com/1127209/169787572-711d3894-7155-4aea-95d4-cef65b97cc25.png)

Apparently the bad value is the string `<nil>`